test/spec/grammar/compatibility_test.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-new, no-unused-expressions */
+const expect = require('chai').expect;
+const parse = require('../../../lib/dsl/poc/api').parse;
+const fs = require('fs');
+const path = require('path');
+const _ = require('lodash');
+
+describe('Chevrotain Migration compatibility harness', () => {
+  describe('Can Parse all the existing JDL samples in the test folder', () => {
+    const samplesDir = path.resolve(__dirname, '../../test_files/');
```

Comment:
Should be `path.join('..', '..', 'test_files')`

Reply:
If I do not use the __dirname it would be susceptible to failing due to different CWD no?